packages/ds-healthcare-gov/src/components/Header/__snapshots__/defaultMenuLinks.test.js.snap

```diff
@@ -13,6 +13,7 @@ Object {
     },
     Object {
       "ariaLabel": "CuidadoDeSalud.gov en Español",
+      "className": undefined,
```

Comment:
Having these here is kind of noisy, but I don't think there's any way they'll hurt anything. Using code to omit them would make the component code less legible, so this is a fine tradeoff.